cloud/wfm/src/executor/ExecutorRepository.ts

```diff
@@ -0,0 +1,6 @@
+import ProcessInstance from '../process-instance/ProcessInstance';
+interface ExecutorRepository {
+  saveInstance(instance: ProcessInstance): Promise<ProcessInstance>;
```

Comment:
Getting error that promise is not defined. Is there any global for this. How this works?